azure-stack/hci/manage/manage-secure-baseline.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Manage baseline security settings on Azure Stack HCI, version 23H2 (preview)
+description: Learn about managing baseline security settings available for new deployments of Azure Stack HCI, version 23H2 (preview).
```

Comment:
Change to: Learn how to manage baseline security settings available for Azure Stack HCI, version 23H2 (preview).

Reply:
Done > updated.